### PR TITLE
upd(adapta-gtk-theme-git): add `pkgver`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

File: packages/adapta-gtk-theme-git/adapta-gtk-theme-git.pacscript [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 }
 
 build() {
-        make -j$(nproc)
+  make -j"$(nproc)"
 }
 
 install() {
-          sudo make install DESTDIR=/usr/src/pacstall/adapta-gtk-theme
+  sudo make install DESTDIR="/usr/src/pacstall/${gives}"
```
Comment (Henryws) [inline packages/adapta-gtk-theme-git/adapta-gtk-theme-git.pacscript:23]:
```suggestion
  sudo make install DESTDIR="${STOWDIR}/${name}"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/adapta-gtk-theme-git/adapta-gtk-theme-git.pacscript (hunk 1 above)
Comment (WRM-42) [reply packages/adapta-gtk-theme-git/adapta-gtk-theme-git.pacscript]: 1e3a288603144b86a7fd4524c4f63d84493d0ecd

**Review by Henryws — CHANGES_REQUESTED** [review]
Oops, look at the CI

**Comment by WRM-42** [comment]
@Henryws Adapta has not been maintained since at least 2018 by looking at the commits log. Wouldn't it be better to remove the package completely?

**Comment by Henryws** [comment]
> @Henryws Adapta has not been maintained since at least 2018 by looking at the commits log. Wouldn't it be better to remove the package completely?

I've still seen people use it, so we should keep it.

**Review by Henryws — CHANGES_REQUESTED** [review]
Check the CI logs; something to do with a newline version

**Review by Henryws — APPROVED** [review]

**Pull request merged**